third_party/libvpx/BUILD.gn

```diff
@@ -108,7 +112,7 @@ if (current_cpu == "x86" || (current_cpu == "x64" && !is_msan)) {
     }
 
     if (!use_cobalt_customizations) {
-      defines = [ "CHROMIUM" ]
+    defines = [ "CHROMIUM" ]
```

Comment:
Do you know what this does and whether excluding it is (still) necessary?

Reply:
Discussed offline